fe4u-boilerplate-main/src/html/Task2.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="description" content="Лабораторная робота №1 с 4 подпунктами">
+    <meta name="msapplication-tap-highlight" content="no">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Project</title>
+</head>
+<style>
+    *{
+        box-sizing: border-box;
+    }
+    div.card_lecturer {
```

Comment:
You may use `<style>` inside `html` only in case when you make email markups.
For all the other cases all styles should be in CSS files, not here.